template projects/blank/0b300310-455a-4a6b-b4b6-de367b5c231d.json

```diff
@@ -1,10 +1,9 @@
 {
   "attributes": {
     "root": true,
-    "template": true,
-    "locked": true
+    "template": true
   },
-  "description": "Project root",
+  "description": "Default project template. It contains a Starter and a Logger node.",
```

Comment:
The text is a bit too long it gets cut on the welcome screen.
Maybe: "Default project template. Just a Starter and a Logger node.